govc/library/create.go

```diff
@@ -49,6 +49,7 @@ func (cmd *create) Register(ctx context.Context, f *flag.FlagSet) {
 	f.StringVar(&cmd.sub.SubscriptionURL, "sub", "", "Subscribe to library URL")
 	f.StringVar(&cmd.sub.UserName, "sub-username", "", "Subscription username")
 	f.StringVar(&cmd.sub.Password, "sub-password", "", "Subscription password")
+	f.StringVar(&cmd.sub.SslThumbprint, "ssl-thumbprint", "", "SSL Tumbprint")
```

Comment:
Thanks @serokles , let's name the flag just `"thumbprint"` without the `ssl-` prefix.  We use that in a few other places, such as the `host.add`, `host.reconnect` and `cluster.add` commands:
https://github.com/vmware/govmomi/blob/7b110ddb44f68a7223d9d6d9e5b1ef8ede929701/govc/flags/host_connect.go#L56

There's a typo in the description too `"Tumbprint"`